### DISCUSSION TITLE
SelectList multiple with toggle selection on items

### DISCUSSION THREAD

**Discussion by damianog** [Megabit/Blazorise#4404] [Q&A]
Is it possible to select/unselect single items on SelectList with Multiple parameter without the need of ctrl key pressed?

**Comment by David-Moreira** [comment]
You can drag click. What would you propose?

**Reply by damianog** [reply]
@David-Moreira I know, in a mobile system the control shows a list with a checkbox to the right that allow to tap single elements.
But in a desktop browser if I want to select single non continuous items I must use the ctrl key also.

**Comment by stsrki** [comment]
Note for self: https://stackoverflow.com/questions/8641729/how-to-avoid-the-need-for-ctrl-click-in-a-multi-select-box-using-javascript
